docs/index.md

```diff
@@ -57,4 +57,65 @@ Get started by [installing](install) this plugin and also refer to the [Release
 
 **NOTE:** The documentation is always up to date with latest version (or master). If you are on old version please refer to the code.
 
+## Automate configuring via Jenkins Script Console
```

Comment:
[config](https://github.com/jenkinsci/jira-steps-plugin/blob/master/docs/pages/general/config.md) is the right place for this, would you mind moving this section there. Thanks.